core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1166,6 +1166,27 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
         fastPassServiceConstructor(childContext, dataSource)
           .syncFastPassesForUserInWorkspace(destWorkspaceContext)
       )
+
+      _ <- Future {
+        workspaceManagerDAO.cloneWorkspace(
+          sourceWorkspaceId = sourceWorkspaceContext.workspaceIdAsUUID,
+          workspaceId = destWorkspaceContext.workspaceIdAsUUID,
+          displayName = destWorkspaceContext.name,
+          spendProfile = None,
+          billingProjectNamespace = destWorkspaceContext.namespace,
+          ctx
+        )
+      }.recoverWith { case e: ApiException =>
+        if (e.getCode != StatusCodes.NotFound.intValue) {
+          logger.warn(
+            s"Unexpected failure cloning workspace (while cloning in Workspace Manager) [sourceWorkspaceId=${sourceWorkspaceContext.workspaceId}, destWorkspaceId=${destWorkspaceContext.workspaceId}]. Received ${e.getCode}: [${e.getResponseBody}]"
```

Comment:
Nit: is it worth including in the message that it is a Rawls stage workspace?

Reply:
Yes, added